src/Evolution/Systems/ScalarWave/Tags.hpp

```diff
@@ -30,4 +31,56 @@ struct Phi : db::SimpleTag {
   using type = tnsr::i<DataVector, Dim, Frame::Inertial>;
   static std::string name() noexcept { return "Phi"; }
 };
+
+namespace Tags {
+struct ConstraintGamma2 : db::SimpleTag {
+  using type = Scalar<DataVector>;
+  static std::string name() noexcept { return "ConstraintGamma2"; }
+};
+
+// @{
```

Comment:
(just picking some line at random)

Something I've generally found very frustrating with all our evolution systems is that they don't have the evolution equations documented in the namespace. What do you think about documenting the evolution equations with gamma2 and all as part of the `ScalarWave` namespace? Right now I don't think gamma2 is really documented.

Reply:
Happy to do this. What would be the right place to add this doc if I want it to show up at https://spectre-code.org/namespacescalarwave @nilsdeppe ?

Reply:
The namespace is documented in the `System.hpp` file, so that would be the right place :)

Reply:
I've added a short description to `System.hpp` for now. Let me know if you want any augmentations/ changes.